quipucords/api/fact/util.py

```diff
@@ -33,6 +33,26 @@
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 
+def build_sources_from_tasks(tasks):
```

Comment:
Not a great name. I know what you mean here, but its more like `build_fact_collection_data_from_tasks()`

Reply:
Its not quite that either.